ci/pr-pipeline.yml

```diff
@@ -57,7 +57,8 @@ jobs:
                 
                 export PATH="$PATH:/usr/local/go/bin"
                 export GOPATH=/go
-                export GO_VERSION=1.19
+                export GO_VERSION=1.21.5
+                export GO_SHA256="e2bc0b3e4b64111ec117295c088bde5f00eeed1567999ff77bc859d7df70078e go${GO_VERSION}.linux-amd64.tar.gz"
```

Comment:
Could you also add the checkshum validation here?

https://github.com/instana/instana-agent-operator/blob/ff74c926419157ddac4978fe18d7e06762adbce7/ci/pipeline.yaml#L557